scripts/jenkins/nightly-report.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+CURRENT_TIMESTAMP=$(date +%s)
+LAST_WEEK_TIMESTAMP=$(( CURRENT_TIMESTAMP - 86400*7 )) # only interested in builds not older than 7 days
+ERROR_STRING="Error 404"
+declare -a JOBS=("rhmi-install-addon-flow" "rhmi-install-master")
+declare -i buildNumber
+declare -i lastBuildNumber
+declare -i buildTimestamp
+declare buildUrl
+declare buildDate
+declare buildResult
+
+getDateString() {
+  case "$OSTYPE" in
+    darwin*)   buildDate=$(date -r $buildTimestamp +%Y-%m-%d) ;;
+    linux*)  buildDate=$(date -d @$buildTimestamp +%Y-%m-%d) ;;
+  esac
+}
+
+rm -f report.txt && touch report.txt
+echo -e "Nightly pipelines results for week ending at $(date +%Y-%m-%d)\n" > report.txt
+for job in "${JOBS[@]}";
+do
+  echo -e "Obtaining nightly pipelines results for $job..."
+  echo -e "Pipeline: $job\n" >> report.txt
+  
+  lastBuildNumber=$(curl $JENKINS_URL/$job/api/json --silent | jq -r .builds[0].number)
+  
+  for ((buildNumber=$lastBuildNumber; buildNumber > 0; buildNumber--))
```

Comment:
Can it also check how many builds there are in the api response, so that it does not need to go to `0`, but something like `lastBuildNumber - numOfBuilds`?

Reply:
that's a good idea. gonna implement it